plotly/offline/offline.py

```diff
@@ -227,12 +227,12 @@ def _plot_html(figure_or_data, config, validate, default_width,
     plotly_platform_url = plotly.plotly.get_config().get('plotly_domain',
                                                          'https://plot.ly')
     if (plotly_platform_url != 'https://plot.ly' and
-            link_text == 'Export to plot.ly'):
+            config['linkText'] == 'Export to plot.ly'):
 
         link_domain = plotly_platform_url\
             .replace('https://', '')\
             .replace('http://', '')
-        link_text = link_text.replace('plot.ly', link_domain)
+        link_text = config['linkText'].replace('plot.ly', link_domain)
         config['linkText'] = link_text
```

Comment:
It's plotting now 🎉
However, the link_text remains: 'Export to plot.ly' while it looks like it should be replaced with the domain